### PR TITLE
chore: postgres config

### PR DESCRIPTION
**Pull request by asdfryan**
## Test Plan

[Describe test plan here]

## Deployment instructions

[Add any special deployment instructions here]

### CODE REVIEW

File: packages/types/destination.ts [hunk 1]
```diff
@@ -8,46 +8,47 @@ type BaseDestinationUpdateParams = BaseDestination;
...
-  config: object; // TODO: define this
+  config: PostgresConfig;
 };
 export type PostgresDestinationUpdateParams = BaseDestinationUpdateParams & {
   type: 'postgres';
-  config: object; // TODO: define this
+  config: PostgresConfig;
+};
+
+export type PostgresConfig = {
```
Comment (albertyfwu) [inline packages/types/destination.ts:45]: where does the customer specify the schema we should write to? or should we use schema to encode `crm`, `hris`, etc.?

Comment (asdfryan) [reply packages/types/destination.ts:45]: We hadn't decided on the shape yet -- as far as I know it's going to be not configurable anyway (either common model, or json) so I'm not sure we'd expose it anyway

Comment (albertyfwu) [reply packages/types/destination.ts:45]: by schema i mean postgres `schema`, like `public`

Comment (asdfryan) [reply packages/types/destination.ts:45]: I see. I guess I can add this (future PR)

**Review by albertyfwu — APPROVED** [review]

**Pull request merged**